include/SDL3/SDL_render.h

```diff
@@ -929,6 +931,40 @@ extern SDL_DECLSPEC SDL_Renderer * SDLCALL SDL_GetRendererFromTexture(SDL_Textur
  */
 extern SDL_DECLSPEC bool SDLCALL SDL_GetTextureSize(SDL_Texture *texture, float *w, float *h);
 
+/**
+ * Set the palette used by a texture.
+ *
+ * A single palette can be shared with many textures.
```

Comment:
From the testpalette example, it looks like the palette is copied.
(The palette is destroyed immediately after creating a texture)
Don't you end up with N palettes for N textures?